lib/upload_creator.rb

```diff
@@ -57,9 +57,6 @@ def initialize(file, filename, opts = {})
         true
       end
     )
-
-    # TODO (martin) Validate @opts[:type] to make sure only blessed types are passed
```

Comment:
This is not relevant, the type is not actually stored anywhere and we test it explicitly as needed